types/express/index.d.ts

```diff
@@ -1,6 +1,7 @@
 // Type definitions for Express 4.17
 // Project: http://expressjs.com
 // Definitions by: Boris Yankov <https://github.com/borisyankov>
+//                 China Medical University Hospital <https://github.com/CMUH>
```

Comment:
It looks like this is an organization, not an individual account, so adding it here will have no effect.

Reply:
Yes, it's fine. This contribution is own by this organization, not only me.